packages/ubuntu_wizard/lib/src/l10n/ubuntu_pt.arb

```diff
@@ -1,5 +1,6 @@
 {
   "@@locale": "pt",
+  "countryCode": "PT",
 
-  "languageName": ""
+  "languageName": "Português"
```

Comment:
The language name was originally left empty to ensure the language wouldn't be displayed in the UI, because there currently are no translations for it (only for Brazilian Portuguese).
It feels a bit weird that I can select Portuguese and the UI is 100% English.

However my understanding is that we're heading towards a list of languages that reflects the availability of langpacks, not the current translations for the app itself. So I suppose it doesn't hurt to keep this entry?

Reply:
A mass import of translations from Ubiquity will follow. (with help of https://github.com/canonical/ubuntu-desktop-installer/pull/463)